src/edu/ucsb/cs56/w15/drawings/parsahadidi/SimpleGui1.java~

```diff
@@ -0,0 +1,27 @@
+package edu.ucsb.cs56.w15.drawings.parsahadidi;
+import javax.swing.*;
+
+/** SimpleGui1 comes from Head First Java 2nd Edition p. 355.
+      It illustrates a simple GUI with a Button that doesn't do anything yet.
+
+     @author Head First Java, 2nd Edition p. 355
+     @author P. Conrad (who only typed it in and added the Javadoc comments)
+     @author Parsa Hadidi
+     @version CS56, Spring 2013, UCSB
```

Comment:
clean your directory of all temporary ~ and # files before submitting pull request